packages/evolution-common/src/services/widgets/WidgetConfig.ts

```diff
@@ -238,7 +240,7 @@ export type InputMapFindPlaceType = InputMapType & {
     updateDefaultValueWhenResponded?: boolean;
 };
 
-export type QuestionWidgetConfig = {
+export type BaseQuestionType = {
     type: 'question';
     twoColumns?: boolean;
     joinWith?: string;
```

Comment:
What is joinWith by the way?

Reply:
It's a way to visually merge 2 related (like the email and phone number widgets when the "agree to be contacted" is set. Like  this:

![Capture d’écran du 2024-10-24 15-32-20](https://github.com/user-attachments/assets/c421ed8d-66eb-40f5-a457-ad009b37d4c0)

(but yeah, it's "poorly" documented...)